packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -1091,6 +1103,13 @@ export class TestObjectProviderWithVersionedLoad implements ITestObjectProvider
 		this._loaderContainerTracker.reset();
 		this._loaderContainerTracker = new LoaderContainerTracker(syncSummarizerClients);
 	}
+
+	/**
+	 * {@inheritDoc ITestObjectProvider.resetDocumentServiceFactory}
+	 */
+	public resetDocumentServiceFactory(): void {
```

Comment:
I am not sure about this. This will need deeper review with people who are famiiar with test object provider. We should only add generic APIs here not something needed by individual test validating very specific scenarios.

Reply:
Removed